help/de/Help_ch_Transactions.xml

```diff
@@ -58,395 +53,439 @@ Translators:
+            oder verwenden Sie die Schaltfläche <guilabel>Wechselkurs abrufen</guilabel>.
           </para>
         </listitem>
 
         <listitem>
-          <para>Select the <guilabel>To Amount</guilabel> radio button if you want to enter the amount of the
-            <emphasis>To Account</emphasis> split.
+          <para>Wählen Sie das Optionsfeld <guilabel>Zum Betrag</guilabel>, wenn Sie den Betrag für den
+            <emphasis>Buchen nach</emphasis> Buchungsteil angeben wollen.
           </para>
         </listitem>
 
         <listitem>
-          <para>Enter the price/exchange rate or the amount in the numeric entry corresponding to radio button you
-            selected. <application>&appname;</application> will calculate the other amount for you
-            when you exit the field. If it's enabled you can use the <guilabel>Fetch Rate</guilabel>
-            button to retrieve a current quote.
+          <para>Geben Sie den Wechselkurs oder den Betrag in das numerische Feld ein, das dem von Ihnen gewählten
+            Optionsfeld entspricht. &app; wird den anderen Betrag für Sie berechnen wenn Sie das
+            Feld verlassen. Wenn aktiviert, können Sie mit der Schaltfläche <guilabel>Wechselkurs
+            abrufen</guilabel> den aktuellen Kurs aus der Datenbank verwenden.
           </para>
         </listitem>
       </itemizedlist>
 
       <note>
-        <para>If you have created a price in the <guilabel>Price Editor</guilabel> (see
-          <xref
-      linkend="invest-stockprice-manual"/>) for the current date quote retrieval
-          will not update that price. You must first delete the manually-created price in the
-          <guilabel>Price Editor</guilabel> before you
-          <keycombo action="click">
-            <mousebutton>Button1</mousebutton>
-          </keycombo>
-          on the <guibutton>Fetch Rate</guibutton> button.
+        <para>Wenn Sie einen Kurs für das aktuelle Datum in der <guilabel>Kurs-Datenbank</guilabel> eingegeben
+          haben (siehe <xref
+      linkend="invest-stockprice-manual"/>) wird der Abruf des
+          Wechselkurs den Kurs nicht aktualisieren. Sie müssen zuerst den manuell erstellten Kurs
+          in der <guilabel>Kurs-Datenbank</guilabel> entfernen bevor Sie die <guibutton>Wechselkurs
+          abrufen</guibutton> Schaltfläche betätigen.
         </para>
       </note>
 
       <note>
-        <para>The <guibutton>Fetch Rate</guibutton> button will immediately create or replace a price in the
-          <guilabel>Price Editor</guilabel> unless there is an existing manually-created price for
-          today.
+        <para>Die Schaltfläche <guibutton>Wechselkurs abrufen</guibutton> erzeugt oder ersetzt sofort einen Kurs
+          in der <guilabel>Kurs-Datenbank</guilabel>, es sei denn, es wurde ein Wechselkurs für
+          heute manuell eingetragen.
```

Comment:
Stimmt wohl nicht mehr ganz so. Es gab irgendwann größere Änderungen am Design der Kursdatenbank.

Jede Währungs- oder Wertpapier-**Transaktion speichert** ihren individuellen Kurs. Diese können auch in der **Kursdatenbank angezeigt** werden. Zusätzlich kann die Kursdatenbank/Tag einen online abgerufenen oder manuell eingegebenen Kurs speichern (und natürlich anzeigen).

Was jetzt in welcher Situation genau passiert, solltest du noch mal überprüfen und im Zweifelsfall jralls fragen.

Reply:
@fellen: ich hab es gerade ausprobiert. Ein manuelle eingetragener Kurs wird mit dem Kursabruf ersetzt.

Reply:
Dann eben: 
Einen Eintrag pro Tag, wobei manuelle Einträge von Kursabfragen am selben Tag überschrieben werden.
oder so ähnlich.

Jedenfalls sollte man das klarer formulieren.
(Nicht irritiert sein, falls wir das schon hatten. Ich gehe gerade einfach meine todo-Liste durch.)